redbiom/commands/summarize.py

```diff
@@ -74,17 +80,70 @@ def summarize_metadata(descending):
         click.echo("%s\t%s" % (idx, val))
 
 
+def _summarize_id(context, category, id):
+    """Summarize the ID over the category"""
+    from redbiom.summarize import category_from_observations
```

Comment:
I think this is the only place where we have: `from redbiom.summarize import category_from_observations` vs. `import redbiom.summarize` and then call `redbiom.summarize.category_from_observations`, why?